hack/agent_installer/start_db.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
This script will be invoked by the systemd service to start db container in the agent installer.